overlord/snapstate/backend/snapdata.go

```diff
@@ -120,6 +120,10 @@ func snapCommonDataDirs(snap *snap.Info, opts *dirs.SnapDirOptions) ([]string, e
 		return nil, err
 	}
 
+	// then the root user's common data dir
+	rootCommon := snap.UserCommonDataDir(filepath.Join(dirs.GlobalRootDir, "/root/"), opts)
+	found = append(found, rootCommon)
```

Comment:
does this need a unit test change?

Reply:
Yes, thanks for pointing it out. Added to an existing test now